spec/system/school/courses_index_spec.rb

```diff
@@ -261,6 +261,10 @@ def file_path(filename)
         'Edit Curriculum',
         href: curriculum_school_course_path(course_1)
       )
+      expect(page).to have_link(
+        'Clone Course',
+        href: clone_school_course_path(course_1)
+      )
```

Comment:
The spec currently checks for the presence of a link. This really should be _clicking_ the link instead of looking for it, and then checking for the presence of a copy afterwards.